src/gc/impl/conservative/gc.d

```diff
@@ -4087,7 +4087,7 @@ debug (LOGGING)
             printf("    p = %p, size = %lld, parent = %p ", p, cast(ulong)size, parent);
             if (file)
             {
-                printf("%s(%u)", file, line);
+                printf("%s(%zu)", file, line);
```

Comment:
%z doesn't work with the older VS runtimes (before VS2015 IIRC).

Reply:
We're kinda hosed, then. Why are we supporting 5 year old VS?

Reply:
The `z` format was standardized in 1999, 21 years ago.

Reply:
%lld and a cast to long is usually used. 

> Why are we supporting 5 year old VS?

mostly because the autotester uses VS2013 and the release process uses VS2010 (hopefully updated soon, but it seems Martin doesn't want to go further than VS2015).

Reply:
@MartinNowak what do you think?

Reply:
(Note that I added hundreds of `zd` and `td` formats to the test suite to resolve those errors.)

Reply:
Maybe the compiler should not be too pedantic: it could only warn about uint and size_t mismatches if it actually makes a difference in the ABI: for example with x64, ints are passed as 64-bit values in registers or on the stack, too. Not sure if the higher bits are allowed to contain garbage, though.

Reply:
It makes a difference with 32 <=> 64 bit compiles.

Reply:
No idea what you mean. In 32-bit processes there is no size-difference between int and size_t, and in 64-bit processes, single integer arguments are always passed by occupying 64-bit.